### PR TITLE
[PM-166] [PM-198] - Add Event Logs for CLI Actions

### PR DESCRIPTION
**Pull request by ttalty**
## Type of change

```
- [ ] Bug fix
- [X] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Identify CLI events from desktop events for existing logging. Add additional logging for CLI events

## Code changes

- **messages.json:** Added the en locale entry for the CLI to display in the event logs correctly
- **device-type.enum.ts:** WindowsCLI, MacOsCLI, and LinuxCLI entries created
- **event.service.ts:** Get app info entries for new CLI device types
- **api.service.ts:** New CLI entries added to isDesktopClient
- **bw.ts:** EventCollectionService and EventUploadService added and created
- **get.command.ts:** EventCollectionService added and collect runs on getting a cipher
- **list.command.ts:** EventCollectionService added and collect runs for each cipher viewed
- **export.command.ts:** EventCollectionService added and collect runs when client exports the vault
- **send.command.ts:** EventCollectionService added to the initialization of the required commands
- **vault.command.ts:** EventCollectionService added to the initialization of the required commands
- **serve.command.ts:** EventCollectionService added to the initialization of the required commands
- **cli-platform-utils.service.ts:** getDevice modified to look for new CLI device types and getDeviceString replaces 'cli' instead of 'desktop'
-

### CODE REVIEW

**Review by aj-rosado — DISMISSED** [review]
Great work on this 💯 
Changes are looking good! 🎉

File: apps/web/src/app/core/event.service.ts [hunk 1]
```diff
@@ -486,6 +486,12 @@ export class EventService {
         return ["bwi-globe", this.i18nService.t("webVault") + " - IE"];
       case DeviceType.Server:
         return ["bwi-server", this.i18nService.t("server")];
+      case DeviceType.WindowsCLI:
+        return ["bwi-windows", this.i18nService.t("cli") + " - Windows"];
+      case DeviceType.MacOsCLI:
+        return ["bwi-apple", this.i18nService.t("cli") + " - macOS"];
+      case DeviceType.LinuxCLI:
+        return ["bwi-linux", this.i18nService.t("cli") + " - Linux"];
```
Comment (MGibson1) [inline apps/web/src/app/core/event.service.ts:494]: These should all probably use the `bwi-cli` icon

**Review by MGibson1 — CHANGES_REQUESTED** [review]
There are many more actions with events that are possible in the CLI, is this meant to be exhaustive or just a start for viewing ciphers and exports? Why those two?

File: apps/web/src/app/core/event.service.ts (hunk 1 above)
Comment (ttalty) [reply apps/web/src/app/core/event.service.ts]: Changed the icons to use bwi-cli

**Review by MGibson1 — APPROVED** [review]
Thanks, this is a long time coming 🎉

**Review by aj-rosado — APPROVED** [review]
Looking good! 🎉

**Pull request merged**